index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Faker Person Portraits Grid</title>
+    <link rel="icon" href="logo.svg" sizes="any" type="image/svg+xml" />
```

Comment:
Maybe instead of duplicating the logo here reuse the one for our homepage?

Reply:
done